build-plugins/src/main/kotlin/aws/sdk/kotlin/gradle/kmp/IdeUtils.kt

```diff
@@ -24,9 +24,7 @@ val HOST_NAME = when {
     else -> error("Unknown os name `$OS_NAME`")
 }
 
-// TODO - enable real logic when ready to add additional targets
-// val Project.COMMON_JVM_ONLY get() = IDEA_ACTIVE && properties["aws.kotlin.ide.jvmAndCommonOnly"] == "true"
-val Project.COMMON_JVM_ONLY get() = true
+val Project.COMMON_JVM_ONLY get() = IDEA_ACTIVE && properties["aws.kotlin.ide.jvmAndCommonOnly"] == "true"
```

Comment:
**suggestion**: We don't have to do this in this PR but I think we should also end up adding boolean properties like `kmpNative`, `kmpJs`, kmpJvm` at some point to allow more precise disabling of certain targets.